en/06_Project_Governance/03_Maintainer_Guidelines.md

```diff
@@ -21,6 +21,10 @@ Refer to the [triage and peer review](/contributing/triage_resources/) for infor
 
 ## Maintainer Roles
```

Comment:
There are four types of maintainer roles, including refiners below.

I don't think that list or table is necessary, since the headings already serve that purpose.

Reply:
I really don't see a need for it but I'll add it to avoid peer review ping pong